srcalc/src/main/java/gov/va/med/srcalc/domain/variable/PopulatedVariableGroup.java

```diff
@@ -20,7 +20,7 @@
      * @throws IllegalArgumentException if the given list is empty or if it
      * contains Variables with different groups
      */
-    public PopulatedVariableGroup(final List<Variable> variables)
+    public PopulatedVariableGroup(final List<? extends Variable> variables)
```

Comment:
Why the change here? Were you specifically trying to exclude any Variable types? I'm a bit fuzzy on the reasoning for "? extends" because I've never actually seen it used in code.